### PR TITLE
Add CRI to glossary

### PR DESCRIPTION
**Pull request by sftim**
I've tagged this glossary entry as “fundamental”.
I did wonder if it's worth adding a new tag for glossary entries to do with containers. What do people think?

### CODE REVIEW

**Comment by poothia** [comment]
lgtm

File: content/en/docs/reference/glossary/cri.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+full_link: https://kubernetes.io/docs/concepts/overview/components/#container-runtime
+short_description: >
+    An API for container runtimes to integrate with kubelet
+
+
+aka:
+tags:
+- fundamental
+---
+The container runtime interface (CNI) is an an API for container runtimes
```
Comment (mistyhacks) [inline content/en/docs/reference/glossary/cri.md]: CRI, right?

Comment (sftim) [reply content/en/docs/reference/glossary/cri.md]: This is why we have reviewers. `:gender_neutral_facepalm:`

Thanks @mistyhacks!

Comment (sftim) [reply content/en/docs/reference/glossary/cri.md]: Fixed & force-pushed (with a rebase against `master`)

**Pull request merged**